financial-connections/src/main/java/com/stripe/android/financialconnections/FinancialConnectionsSheetViewModel.kt

```diff
@@ -60,16 +46,19 @@ internal class FinancialConnectionsSheetViewModel @Inject constructor(
      * as well as the success and cancel callback URLs to verify.
      */
     private fun fetchManifest() {
-        viewModelScope.launch {
-            kotlin.runCatching {
-                generateFinancialConnectionsSessionManifest(
-                    clientSecret = starterArgs.configuration.financialConnectionsSessionClientSecret,
-                    applicationId = applicationId
-                )
-            }.onFailure {
-                onFatal(it)
-            }.onSuccess {
-                openAuthFlow(it)
+        withState { state ->
```

Comment:
state is read via `withState`. It ensures all the pending state changes are commited.